src/hf_hydrodata/point.py

```diff
@@ -2124,47 +2170,66 @@ def _get_data_sql(conn, site_list, var_id, *args, **kwargs):
                 options["date_end"], "%Y-%m-%d"
             ).year
 
-    if ("date_start" not in options) and ("date_end" not in options):
-        date_query = """"""
-        param_list = [min_num_obs]
-    elif ("date_start" not in options) and ("date_end" in options):
-        date_query = """ WHERE w.date <= ?"""
-        param_list = [options["date_end"], min_num_obs, options["date_end"]]
-    elif ("date_start" in options) and ("date_end" not in options):
-        date_query = """ WHERE w.date >= ?"""
-        param_list = [options["date_start"], min_num_obs, options["date_start"]]
-    elif ("date_start" in options) and ("date_end" in options):
-        date_query = """ WHERE w.date >= ? AND w.date <= ?"""
-        param_list = [
-            options["date_start"],
-            options["date_end"],
-            min_num_obs,
-            options["date_start"],
-            options["date_end"],
-        ]
+    # These variables have w.date in the database to be able to filter on
+    if var_id in (5, 25):
+        if ("date_start" not in options) and ("date_end" not in options):
+            date_query = """"""
+            param_list = [min_num_obs]
+        elif ("date_start" not in options) and ("date_end" in options):
+            date_query = """ WHERE w.date <= ?"""
+            param_list = [options["date_end"], min_num_obs, options["date_end"]]
+        elif ("date_start" in options) and ("date_end" not in options):
+            date_query = """ WHERE w.date >= ?"""
+            param_list = [options["date_start"], min_num_obs, options["date_start"]]
+        elif ("date_start" in options) and ("date_end" in options):
+            date_query = """ WHERE w.date >= ? AND w.date <= ?"""
```

Comment:
Same comment here about `""` vs `""" """` Seems like a single quotes would work just fine for this

Reply:
Same as above - updated.